code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -274,6 +324,7 @@
 			speaker_name = H.real_name
 		rendered_message = speech.render_message()
 		for(var/mob/living/L in player_list)
```

Comment:
Personally I would get the Culto's faction, and loop over the roles in that faction rather than over the player list
But this is marginal

Reply:
Will do that once different cults are a thing.